### ISSUE TITLE
Example proto file from official site cannot be parsed (use of package)

### ISSUE THREAD

**Issue by iameugenejo** [appnexus/pyrobuf#9]
Tried building with the example at https://developers.google.com/protocol-buffers/docs/pythontutorial

``` proto
package tutorial;

message Person {
  required string name = 1;
  required int32 id = 2;
  optional string email = 3;

  enum PhoneType {
    MOBILE = 0;
    HOME = 1;
    WORK = 2;
  }

  message PhoneNumber {
    required string number = 1;
    optional PhoneType type = 2 [default = HOME];
  }

  repeated PhoneNumber phone = 4;
}

message AddressBook {
  repeated Person person = 1;
}

```

and I get this error:

``` bash
parsing messages/test.proto
Traceback (most recent call last):
  File "gen_all_messages.py", line 33, in <module>
    gen_all_messages()
  File "gen_all_messages.py", line 21, in gen_all_messages
    msgdef = parser.parse_from_filename(spec)
  File "/Users/ejo-mac15/git/pyrobuf/parse_proto.py", line 162, in parse_from_filename
    return self.parse(s)
  File "/Users/ejo-mac15/git/pyrobuf/parse_proto.py", line 140, in parse
    for token in tokens:
  File "/Users/ejo-mac15/git/pyrobuf/parse_proto.py", line 134, in tokenize
    raise Exception("Unexpected character '%s' in line %d: '%s'" % (s[pos], line, s[pos-10:pos+10]))
Exception: Unexpected character 'p' in line 1: ''
```

Is this a bug?

**Comment by 3kwa** [comment]
I have a PR pending that fixes that...

**Comment by mmoresco** [comment]
Looks like there's still an issue with nested enums - I'll fix that.

**Comment by 3kwa** [comment]
What would be awesome is the whole tutorial implemented as a collection of tests so we could see where we are at!

**Comment by AndreMiras** [comment]
I've fixed nested enums in PR #68 and the tutorial file seems to work. Please fill free to test and give some feedback.
